src/inject.js

```diff
@@ -1,6 +1,6 @@
 WAPI.waitNewMessages(false, (data) => {
-    console.log(data)
-    data.forEach((message) => {
+    console.log(data);
+    data.forEach(async (message) => {
```

Comment:
this won't await. We'll have to convert the loop into `for(i=0; i<data.length; i++)` style

Reply:
Thank you for pointing the mistake. I have made the changes please review.